pb_model/tests/tests.py

```diff
@@ -181,7 +182,7 @@ def test_auto_fields(self):
             bytes_field=b'123',
             bool_field=True,
             uuid_field=str(uuid.uuid4()),
-
+            any_field=Any(),
```

Comment:
Great to support autogen and testing! I think we could non-empty message and add assertion below to verify the mechanism.

Reply:
I changed it to add the Timestamp created above to inside the any 😄